### PR TITLE
Add utility for deleting entities in a workspace

### PR DESCRIPTION
**Pull request by melissachang**
FYI @malathi13

### CODE REVIEW

File: util/delete-entities.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+def main():
+    if len(sys.argv) != 3:
+        raise ValueError(
+            'Usage: python util/delete-entities.py WORKSPACE_NAMESPACE WORKSPACE_NAME'
+        )
+
+    workspace_namespace = sys.argv[1]
+    workspace_name = sys.argv[2]
+
+    args = type('obj', (object, ), {
```
Comment (bfcrampton) [inline util/delete-entities.py]: I think the standard way to do this is by using a [namedtuple](https://docs.python.org/2/library/collections.html#collections.namedtuple) here:
```
Args = collections.namedtuple('Args', ['project', 'workspace'])
args = Args(workspace_namespace, workspace_name)
```

Though this is pretty neat, didn't realize it was possible this way.

File: util/delete-entities.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+    # entities are sorted by type: participant, participant_set, sample,
+    # sample_set. FireCloud complains if we delete a participant before deleting
+    # associated participant_set/sample/sample_set. By reversing the list, we
+    # won't have this problem. We will delete: sample_set, sample,
+    # participant_set, participant.
+    for entity in reversed(entities):
+        print('Deleting ' + entity)
+        args = type(
+            'obj', (object, ), {
```
Comment (bfcrampton) [inline util/delete-entities.py]: I would probably just extract the namedtuple out to the top level:

```
Args = collections.namedtuple('Args', ['project', 'workspace', 'yes', 'entity_type', 'entity'])
```

Then above:
```
entities = fiss.entity_list(Args(workspace_namespace, workspace_name))
```

and here:
```
entity_split entity.split('\t')
fiss.entity_delete(Args(workspace_namespace, workspace_name, True, entity_split[0], entity_split[1]))
```

Could also use keyword arguments if you think that makes it clearer.

File: util/delete-entities.py (hunk 1 above)
Comment (melissachang) [reply util/delete-entities.py]: Done

File: util/delete-entities.py (hunk 2 above)
Comment (melissachang) [reply util/delete-entities.py]: Done

**Pull request merged**